### PR TITLE
ATR: step creation wizard

### PR DESCRIPTION
**Pull request by eberloso**
### Description:

 Adds documentation about the wizard used to create steps of ATR cases and reformat all the ATR doc. fixing all the titles.

### Tasks:

- [x] Catalan doc.
- [x] Spanish doc.

### Links:

* [atr.md #Assistent de Creació de Passos ATR](http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/ca/base/atr/#assistent-de-creacio-de-passos-atr)
* [atr.md #Assistente de Creación de Passos ATR](http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/es//base/atr/#assistente-de-creacion-de-pasos-atr)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

File: docs/base/atr.md [hunk 1]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
-# Apendix A: Exemple de procés C1
+#### Assistent de Creació de Passos ATR
+
+
+Aquest assistent permet la creació de passos dels casos ATR, tan per un cas com
+de forma massiva. En ambdós situacions el seu funcionament és el mateix:
+
+
+* Seleccionar els casos dels quals es vulgui generar un pas i obrir l'assistent
+(si s'esta dins el formulari de un cas, obrir l'assitent directament).
```
Comment (jaumef) [inline docs/base/atr.md]: `s'esta` → `s'està`
`de un` → `d'un`

File: docs/base/atr.md [hunk 2]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
 S'ha integrat la generació de casos i processo B1 amb les devolucions i els
 talls.
 
 
-# Apendix A: Exemple de procés C1
+#### Assistent de Creació de Passos ATR
+
+
+Aquest assistent permet la creació de passos dels casos ATR, tan per un cas com
+de forma massiva. En ambdós situacions el seu funcionament és el mateix:
```
Comment (jaumef) [inline docs/base/atr.md]: `ambdós` → `ambdues`

File: docs/base/atr.md [hunk 3]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+
+* Seleccionar els casos dels quals es vulgui generar un pas i obrir l'assistent
+(si s'esta dins el formulari de un cas, obrir l'assitent directament).
+
+
+* Es mostrara una finestra amb els següents camps:
```
Comment (jaumef) [inline docs/base/atr.md]: `mostrara` → `mostrarà`

File: docs/base/atr.md [hunk 4]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+    En la fase inicial es mostra quants casos s'han seleccionat.
+
+
+    ![](_static/atr/wizard_pas_atr2.png)
+
+
+* Clicant a "Continuar" s'intentarà crear el pas seleccionat en els casos.
+Un cop fet es mostra una pantalla amb un resum en que s'indica:
+
+  * Numero de casos en que s'ha creat el pas correctament
```
Comment (jaumef) [inline docs/base/atr.md]: `Numero → Número`
(es) `Numero → Número`

identació de 4 espais per llistes anidades

File: docs/base/atr.md [hunk 5]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+  Si abans s'ha desmarcat la opció "Ignorar Repetits", aquest numero serà sempre 0.
+
+  * Numero de casos en que hi ha hagut algún error i no s'ha creat el pas.
+
+  * Informació detallada dels casos en que no s'ha creat el pas.
+
+  ![](_static/atr/wizard_pas_atr1.png)
+
+
+* Finalment tenim la opció de tancar l'assistent o de obrir els casos en que
```
Comment (jaumef) [inline docs/base/atr.md]: `de obrir` → `d'obrir`

File: docs/base/atr.md [hunk 6]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+    ![](_static/atr/wizard_pas_atr2.png)
+
+
+* Clicant a "Continuar" s'intentarà crear el pas seleccionat en els casos.
+Un cop fet es mostra una pantalla amb un resum en que s'indica:
+
+  * Numero de casos en que s'ha creat el pas correctament
+
+  * Numero de casos en que no s'ha creat el pas perquè ja existia en el cas.
```
Comment (jaumef) [inline docs/base/atr.md]: (es) `existia` → `existía`

File: docs/base/atr.md [hunk 7]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
 
 S'ha integrat la generació de casos i processo B1 amb les devolucions i els
 talls.
 
 
-# Apendix A: Exemple de procés C1
+#### Assistent de Creació de Passos ATR
+
+
+Aquest assistent permet la creació de passos dels casos ATR, tan per un cas com
```
Comment (jaumef) [inline docs/base/atr.md]: `tan`→`tant`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

File: docs/base/atr.md [hunk 8]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+* Seleccionar els casos dels quals es vulgui generar un pas i obrir l'assistent
+(si s'està dins el formulari d'un cas, obrir l'assitent directament).
+
+
+* Es mostrarà una finestra amb els següents camps:
 
-# Apendix B: Traducció de codi REE
+    * *Pas*: desplegable amb la llista de pasos que es poden crear per els casos seleccionats.
+
+    * *Ignorar repetits*: amb aquesta opció marcada s'evitarà que es crein pasos repetits en els casos.
```
Comment (jaumef) [inline docs/base/atr.md]: `crein` → `creïn`

File: docs/base/atr.md [hunk 9]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+    Per exemple: si un cas ja té el pas 02 i es selecciona crear passos 02 en el desplegable, no es tornarà a crear el pas.
+
+    * *Info*: cuadre de text amb informació útil sobre el procés de creació de pasos.
```
Comment (jaumef) [inline docs/base/atr.md]: `cuadre` → `quadre`
`pasos` → `passos`

File: docs/base/atr.md (hunk 8 above)
Comment (gdalmau) [inline docs/base/atr.md]: `crein pasos` → `creïn passos`

File: docs/base/atr.md (hunk 9 above)
Comment (gdalmau) [inline docs/base/atr.md]: `cuadre` → `quadre`

File: docs/base/atr.md [hunk 10]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+    En la fase inicial es mostra quants casos s'han seleccionat.
+
+
+    ![](_static/atr/wizard_pas_atr2.png)
+
+
+* Clicant a "Continuar" s'intentarà crear el pas seleccionat en els casos.
+Un cop fet es mostra una pantalla amb un resum en que s'indica:
```
Comment (gdalmau) [inline docs/base/atr.md]: `que` → `què`

File: docs/base/atr.md [hunk 11]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+    * Número de casos en que s'ha creat el pas correctament
```
Comment (gdalmau) [inline docs/base/atr.md]: `que` → `què`

File: docs/base/atr.md [hunk 12]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+    * Número de casos en que no s'ha creat el pas perquè ja existía en el cas.
```
Comment (gdalmau) [inline docs/base/atr.md]: `que` → `què`

File: docs/base/atr.md [hunk 13]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+    Si abans s'ha desmarcat la opció "Ignorar Repetits", aquest numero serà sempre 0.
```
Comment (gdalmau) [inline docs/base/atr.md]: `la opció` → `l'opció`
`numero` → `número`

File: docs/base/atr.md [hunk 14]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+    * Número de casos en que hi ha hagut algún error i no s'ha creat el pas.
```
Comment (gdalmau) [inline docs/base/atr.md]: `que` → `què`
`algún` → `algun`

File: docs/base/atr.md [hunk 15]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+    * Informació detallada dels casos en que no s'ha creat el pas.
```
Comment (gdalmau) [inline docs/base/atr.md]: `que` → `què`

File: docs/base/atr.md [hunk 16]
```diff
@@ -779,20 +835,66 @@ conté s'actúa de la següent forma:
...
+
+  ![](_static/atr/wizard_pas_atr1.png)
+
+
+* Finalment tenim la opció de tancar l'assistent o d'obrir els casos en que
```
Comment (gdalmau) [inline docs/base/atr.md]: `la opció` → `l'opció`
`que` → `què`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_doc_wizard_step_atr/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**